.github/actions/install-dependencies/action.yml

```diff
@@ -27,7 +27,8 @@ runs:
           libtinfo-dev \
           flex \
           libfl-dev \
-          flexc++
+          flexc++ \
+          mingw-w64
```

Comment:
How large is this package? (i.e. how much overhead does this add to the regular builds?) We could also add another step for cross-compilation software.